src/main/java/org/kainos/ea/resources/WebService.java

```diff
@@ -33,4 +36,20 @@ public Response getJobRoles() throws SQLException, DatabaseConnectionException {
             return Response.status(HttpStatus.INTERNAL_SERVER_ERROR_500).build();
         }
     }
+
+    @GET
+    @Path("/job-specification/{id}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getJobSpecification( @PathParam("id") int id ) throws SQLException, DatabaseConnectionException {
+
+        try {
+
+            return Response.ok(jobsService.getJobSpecification( id )).build();
```

Comment:
we should have error handling and exceptions for the id provided not existing in db

Reply:
I don't see DataNotFoundException being thrown by the data or service layers or did I miss it?